runtime/browser/web_application.cc

```diff
@@ -485,13 +484,74 @@ void WebApplication::Terminate() {
   }
 }
 
+void WebApplication::CreateWindow(common::AppControl* appcontrol) {
+  auto window_category = GetWindowCategory(appcontrol);
+  window_.reset(CreateNativeWindow(window_category));
+
+  SetWindowOrientation();
+  SetSplashScreen();
+
+  WebView* view = new WebView(window_.get(), ewk_context_);
+  SetupWebView(view);
+
+  std::unique_ptr<common::ResourceManager::Resource> res =
+      resource_manager_->GetStartResource(appcontrol);
+  view->SetDefaultEncoding(res->encoding());
+
+  STEP_PROFILE_END("OnCreate -> URL Set");
+  STEP_PROFILE_START("URL Set -> Rendered");
+
+  window_->SetContent(view->evas_object());
+  view->LoadUrl(res->uri(), res->mime());
+  view_stack_.push_front(view);
+
+  // TODO(sngn.lee): check the below code location.
+  // in Wearable, webkit can render contents before show window
+  // but Mobile, webkit can't render contents before show window
+  window_->Show();
+  window_->Active();
+}
+
+void WebApplication::RemoveWindow() {
+  ClearViewStack();
+
+  window_->SetContent(NULL);
+
+  delete view_stack_.front();
+  view_stack_.clear();
+}
+
+runtime::NativeWindow::WindowCategory WebApplication::GetWindowCategory(
```

Comment:
GetWindowCategory should implement below pseudo code

static WindowCategory WebApplication::GetWindowCategory(AppControl* appcontrol) {
   if (strcmp(appcontrol->data("always_on_top").c_str(), "true") == 0)
       return NativeWindow::WindowCategory::NOTIFICATION;
   else 
      return WindowCategory::NORMAL;
}